tokio/tests/rt_common.rs

```diff
@@ -601,6 +601,7 @@ rt_test! {
     }
 
     #[test]
+    #[cfg(not(windows))]
```

Comment:
This test can't pass on windows. For now, I'm going to disable it. I will document why before completing the PR. I mostly want to see if CI passes.